pkg/server/handler/common.go

```diff
@@ -38,11 +39,19 @@ type AuthResponse struct {
 	LastSeenAt  *time.Time `json:"last_seen_at,omitempty"`
 }
 
-func NewAuthResponse(info skydb.AuthInfo, accessToken string) AuthResponse {
+func NewAuthResponse(info skydb.AuthInfo, authdata *skydb.AuthData, accessToken string) AuthResponse {
+	username := ""
+	email := ""
+
+	if authdata != nil {
+		username, _ = (*authdata)["username"].(string)
```

Comment:
Sometimes you don’t have the chance to come back and change it later. Plus it you define a function you will be sure there is no where trying to access the value from the `authdata` map if you remove it later.